components/styled/table/src/molecules/table-styles.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  CSS_RESET,
+  TRANSLUCENT_MID_GREY,
+  SCROLLBAR,
+  TRANSLUCENT_BRIGHT_BLUE,
+  BTN_COLOURS,
+  TABLE_COLOURS,
+} from '@ltht-react/styles'
+import styled from '@emotion/styled'
+
+const Container = styled.div`
+  ${CSS_RESET};
+  display: inline-block;
+  max-width: 100%;
+  max-height: 100%;
+  border-radius: 6px;
+`
+const ScrollableContainer = styled.div`
+  ${CSS_RESET};
+  background-color: white;
+  display: inline-block;
+  max-width: 100%;
+  max-height: 100%;
+  border-radius: 6px;
+  overflow: auto;
+  &::-webkit-scrollbar {
+    width: 7px;
+    height: 7px;
+    border: 0;
+  }
+  &::-webkit-scrollbar-thumb {
+    background: ${SCROLLBAR};
+    border-radius: 10px;
+  }
+`
+const StyledTable = styled.table`
+  background-color: white;
+  border-collapse: separate;
+  border-spacing: 0px;
+  border-radius: 6px;
+`
+
+const StyledTableHeader = styled.th<IStyledTableCell>`
+  background-color: ${TABLE_COLOURS.HEADER};
+  border: thin solid ${TABLE_COLOURS.BORDER};
+  font-weight: bold;
+  padding: 1rem;
+
+  ${({ stickyWidth }) =>
+    stickyWidth !== undefined &&
+    `
+    position: sticky !important;
+    left: ${stickyWidth}px;
+    top: 0;
+    z-index: 1;`}
+`
+
+const StyledTableData = styled.td<IStyledTableCell>`
+  border: thin solid ${TABLE_COLOURS.BORDER};
+  white-space: nowrap;
+
+  ${({ stickyWidth }) =>
+    stickyWidth !== undefined &&
+    `
+    position: sticky !important;
+    left: ${stickyWidth}px;
+    top: 0;
+    z-index: 1;`}
+
+  &:first-of-type {
+    background-color: ${TABLE_COLOURS.HEADER} !important;
+  }
+`
+
+const PaginationContainer = styled.div`
+  ${CSS_RESET};
+  margin-top: 5px;
+  display: block;
+`
+const PaginationButtonStyle = `
+padding: 2px 5px;
+background-color: ${TRANSLUCENT_BRIGHT_BLUE};
+color: black;
+border: 1px solid ${TRANSLUCENT_MID_GREY};
+margin: 0 2.5px;
+border-radius: 3px;
+
+&:disabled {
+  background-color: inherit;
+  color: gray;
+  border-color: ${TRANSLUCENT_MID_GREY};
+  pointer-events: none;
+}
+`
+const StyledPaginationPageInput = styled.input`
+  ${CSS_RESET};
+  width: 50px;
+  border: 1px solid gray;
+`
+const StyledPaginationPageSelect = styled.select`
+  ${CSS_RESET};
+  width: 45px;
+  display: inline-block;
+  font-size: 0.9rem;
+  border: 1px solid gray;
+`
+const StyledHideOnMobile = styled.span`
+  @media (max-width: 320px) {
+    display: none;
+  }
+`
+
+const StyledPaginationButtonDiv = styled.div`
+  float: right;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+`
+const StyledPageCountDiv = styled.div`
+  margin-right: 5px;
+  margin-left: 5px;
+  display: flex;
+  font-size: 1.1em;
+  padding: 1px;
+`
+
+const StandardButtonStyle = `
```

Comment:
This one and `PaginationButtonStyle ` return string literals but the others returns JSX Elements - it would be nice to be consistent throughout the file. We could either make them all string literals or all JSX elements - 
If the file is called `table-styles` I'd suggest giving it the `.ts` extension and just return the string literals. 
If we're returning the JSX elements I'd suggest it be named something like `table-styled-components` to be clear what it contains